### DISCUSSION TITLE
Revisión Extraordinaria Programación 1

### DISCUSSION THREAD

**Discussion by andreslb9** [mmasias/23-24-prg1#179] [General]
Hola, hay gente que ha hecho la recuperación y tiene examen el día de la revisión, para que les pudieras cambiar la fecha de esta.
Un saludo

**Comment by mmasias** [comment]
Tienes razón. Dejamos abierta entonces la fecha de revisión a mañana y pasado.

**Reply by andreslb9** [reply]
Hay otra persona que pasado mañana también tiene examen a esa hora, me ha dicho que el jueves sobre las 11 30 si podría
